components/script_bindings/utils.rs

```diff
@@ -579,15 +579,31 @@ impl AsCCharPtrPtr for [u8] {
 pub(crate) unsafe extern "C" fn enumerate_global<D: DomTypes>(
```

Comment:
Mozilla has two things for this:

1. https://searchfox.org/mozilla-central/search?q=symbol:_ZN7mozilla3dom15EnumerateGlobalEP9JSContextN2JS6HandleIP8JSObjectEENS3_13MutableHandleINS3_13StackGCVectorINS3_11PropertyKeyEN2js15TempAllocPolicyEEEEEb&redirect=false
2. https://searchfox.org/mozilla-central/search?q=symbol:_ZN19nsGlobalWindowInner19GetOwnPropertyNamesEP9JSContextN2JS13MutableHandleINS2_13StackGCVectorINS2_11PropertyKeyEN2js15TempAllocPolicyEEEEEbRN7mozilla11ErrorResultE&redirect=false

I think we should link them in some comments somewhere here.